### ISSUE TITLE
Incomplete rate set

### ISSUE THREAD

**Issue by liliana-calderon** [MPIDR/rsocsim#13]
Hi SOCSIM team, 
I am running some simulations using the latest version of rsocsim and I am still getting the following error message in the console for all years (here I am only copying for 2022)

> ------------b118b-load.cpp->load . Sweden_a0b1.sup
> #load.cpp->load 4. Sweden_a0b1.sup
> 18b-load.cpp->load . rates/SWEfert2022
> #load.cpp->load 4. rates/SWEfert2022
> Incomplete rate set, will add rate till MAXUYEARS (death-->1.0; others:0.0) 144
> Incomplete rate set, will add rate till MAXUYEARS (death-->1.0; others:0.0) 144
> 18b-load.cpp->load . rates/SWEmort2022
> #load.cpp->load 4. rates/SWEmort2022
> Incomplete rate set, will add rate till MAXUYEARS (death-->1.0; others:0.0) 12
> Incomplete rate set, will add rate till MAXUYEARS (death-->1.0; others:0.0) 12
> New events generated for all living persons__

 I think this comes from the fact that the MAXUYEARS is 200 but I am using the rates only until 110 years, but I believe this could be confusing for users. 
Is there any way to fix this message or make it more clear? 

Thank you in advance, 

Liliana

**Comment by liliana-calderon** [comment]
Also, the warning messages that Antonino mentioned in a previous issue are still there. 

In readLines(con, n = 1) :
  incomplete final line found on 'sim_results_Sweden_a0b1.sup_24072_/logfile.log'
14: In startSocsimWithFile(supfile, seed, compatibility_mode,  ... :
  can't open  transition history file. Hope that's OK

Should it be possible to fix this? 

Best, 

Liliana

**Comment by tomthe** [comment]
Yes, I can change this. But I have to think a little bit about if we need this message at all or how to phrase it. It is probably fine to omit these messages.
